src/azure-cli/azure/cli/command_modules/managedservices/_help.py

```diff
@@ -0,0 +1,174 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+helps['managedservices'] = """
+    type: group
+    short-summary: Manage the registration assignments and definitions in Azure.
+"""
+
+helps['managedservices definition'] = """
+    type: group
+    short-summary: Manage the registration definitions in Azure.
+"""
+
+helps['managedservices definition assignment'] = """
+    type: group
+    short-summary: Manage the registration assignments in Azure.
+"""
+
+helps['managedservices definition create'] = """
+    type: command
+    short-summary: Creates a new registration definition.
+    long-summary: Creates a new registration definition under the default scope or under the subscription provided.
+    parameters:
+        - name: --name -n
+          short-summary: |
+             The name of the registration definition.
+        - name: --tenant-id
+          short-summary: |
+             The managed by tenant id.
+        - name: --principal-id
+          short-summary: |
+             The principal id.
+        - name: --role-definition-id
+          short-summary: |
+             The role definition id.
+        - name: --description
+          short-summary: |
+             Optional: The friendly description.
```

Comment:
Remove any references to "Required:" or "Optional:". The absence of the CLI's automatic `[Required]` tag implies that something is optional.